mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -356,7 +353,7 @@ function resultsToTables(results, target, uri) {
     tableid = "#" + tableid;
     $(target).html(html);
     $(target).attr('data', uri);
-    //retreive albumcovers
+    //retrieve albumcovers
     for (i = 0; i < coversList.length; i++) {
         getCover(coversList[i][0], target + '-cover-' + coversList[i][1], 'small');
```

Comment:
From what I can see, ```resultsToTables``` only shows images if an 'album header' forms part of the search results:

![skitched-20160221-200155](https://cloud.githubusercontent.com/assets/10268911/13204328/f7d47320-d8d5-11e5-8b47-36536aeaf15f.png)

...and I think that only happens when all of the tracks in the entire album are listed in either the search results or the play queue. Many of the extensions that I am using adds individual tracks to the queue however, so I don't often get to see album covers in the results tables.

![skitched-20160221-200429](https://cloud.githubusercontent.com/assets/10268911/13204344/537dc2da-d8d6-11e5-82b5-08e26e7c7141.png)

Probably nothing wrong with this, just a little surprising in terms of what I would expect personally.